.github/workflows/brakeman-analysis.yml

```diff
@@ -0,0 +1,44 @@
+# This workflow integrates Brakeman with GitHub's Code Scanning feature
+# Brakeman is a static analysis security vulnerability scanner for Ruby on Rails applications
+
+name: Brakeman Scan
+
+# This section configures the trigger for the workflow. Feel free to customize depending on your convention
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  brakeman-scan:
+    name: Brakeman Scan
+    runs-on: ubuntu-18.04
+    steps:
+    # Checkout the repository to the GitHub Actions runner
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    # Customize the ruby version depending on your needs
+    - name: Setup Ruby
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.4'
```

Comment:
`2.3.7`

I tried running OFN with `2.4` the other week, and we had fatal errors.

Reply:
commit message explains this:
https://github.com/openfoodfoundation/openfoodnetwork/pull/6163/commits/dbd41e9d318dc2e3d124053deab6c67c617b812b
I dont think this is running the app in 2.4, just brakeman. github actions doesnt support 2.3.